services/src/handlers/spatial_references.rs

```diff
@@ -0,0 +1,132 @@
+use geoengine_datatypes::{
+    primitives::BoundingBox2D,
+    spatial_reference::{SpatialReference, SpatialReferenceAuthority},
+};
+use serde::{Deserialize, Serialize};
+use warp::Filter;
+
+use crate::error;
+use crate::handlers::{authenticate, Context};
+use crate::users::session::Session;
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct SpatialReferenceSpecification {
+    name: String,
+    spatial_reference: SpatialReference,
+    proj_string: String,
+    extent: BoundingBox2D,
+    axis_labels: Option<(String, String)>,
+}
+
+pub(crate) fn get_spatial_reference_specification_handler<C: Context>(
+    ctx: C,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
+    warp::path!("spatialReferenceSpecification" / String)
+        .and(warp::get())
+        .and(authenticate(ctx))
+        .and_then(get_spatial_reference_specification)
+}
+
+async fn get_spatial_reference_specification(
+    srs_string: String,
+    _session: Session,
+) -> Result<impl warp::Reply, warp::Rejection> {
+    // TODO: get specification from Proj or some other source
```

Comment:
we need a task / or story for this. We also need a service where we can request proj-strings / wkt for the SPatialReference in the BAckend.

Reply:
I added this to the projection story https://trello.com/c/d4RnZiEG